necst/core/recorder.py

```diff
@@ -69,6 +70,11 @@ def append(self, msg, topic_name: str) -> None:
             {"key": name, "type": type_, "value": getattr(msg, name)}
             for name, type_ in fields.items()
         ]
+        for i in range(len(chunk)):
+            if "string" in chunk[i]["type"]:
+                chunk[i]["value"] = chunk[i]["value"].ljust(
+                    int(re.sub(r"\D", "", chunk[i]["type"]))
+                )
```

Comment:
`for` 文のループ変数にインデックスを使うのはどうしても必要な場合のみにしてください。
また、長さを制限しない型指定 `"string"` について `"\D"` (数字以外) を `""` で置き換えた場合、76行目は `int("")` となり、エラーになる気がします。

```suggestion
        for _chunk in chunk:
            if "string" in _chunk["type"]:
                _chunk["value"] = _chunk["value"].ljust(
                    int(re.sub(r"\D", "", _chunk["type"]) or len(_chunk["value"]))
                )
```

Reply:
ご指摘ありがとうございます。修正しました。